suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -648,6 +645,7 @@ export const deviceConnectThunks = createThunk<void, DeviceConnectThunksParams,
         switch (type) {
             case DEVICE.CONNECT:
                 dispatch(deviceActions.connectDevice({ device, settings }));
+                dispatch(startDiscoveryThunk({ device }));
```

Comment:
When device is connected at some point, discovery should start, but this clashes with the authenticity check as explained in the main thread.